src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/ParseTreeBuilder.java

```diff
@@ -5772,4 +5772,61 @@ public static ParseTree treeCustomTagWithDot()
         return javadocContext0;
     }
 
+    public static ParseTree treeInlineCustomJavadocTag()
+    {
```

Comment:
`LeftCurlyCheck` isn't picking this up because we are suppressing everything for these files at https://github.com/checkstyle/checkstyle/blob/master/config/suppressions.xml#L56-L57 .
I will make new issues.